src/main/resources/liquibase/workbench_changelog/22_3_0.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+	<changeSet author="aldrin" id="v22.3.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="one_time_password"></tableExists>
+			</not>
+		</preConditions>
+		<comment>
+			Create table 'one_time_password'
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			CREATE TABLE `workbench`.`one_time_password` (
+				`id`       INT NOT NULL AUTO_INCREMENT,
+				`otp_code` INT NULL,
+				`expires`  DATETIME NULL,
+				PRIMARY KEY (`id`)
+			);
+		</sql>
+	</changeSet>
+	<changeSet author="aldrin" id="v22.3.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="user_device_metadata"></tableExists>
+			</not>
+		</preConditions>
+		<comment>
+			Create table 'user_device_meta_data'
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+		  CREATE TABLE `workbench`.`user_device_meta_data` (
+			  `id`             INT NOT NULL AUTO_INCREMENT,
+			  `userid`         INT NULL,
+			  `device_details` VARCHAR(300) NULL,
+			  `location`       VARCHAR(300) NULL,
+			  `last_logged_in` DATETIME NULL,
+			  PRIMARY KEY (`id`)
+		  );
+		</sql>
+	</changeSet>
+	<changeSet author="aldrin" id="v22.3.0-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="users" columnName="multi_factor_authentication"/>
+			</not>
+		</preConditions>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE `workbench`.`users`
+				ADD COLUMN `multi_factor_authentication` TINYINT NULL DEFAULT 0 AFTER `cdate`;
```

Comment:
@abatac when reading first time, I wasnt hundred percent clear, I think multi_factor_enabled represents better the stored value

Reply:
or a little bit longer.. `multi_factor_authentication_enabled`
it could also be `multi_factor_enabled` with a column `message` for more info

Reply:
I changed the column name to multi_factor_authentication_enabled